src/pubsub/ua_pubsub_reader.c

```diff
@@ -366,11 +905,17 @@ UA_Server_removeDataSetReader(UA_Server *server, UA_NodeId readerIdentifier) {
         return UA_STATUSCODE_BADNOTFOUND;
     }
 
+    if(dataSetReader->configurationFrozen){
+        UA_LOG_WARNING(&server->config.logger, UA_LOGCATEGORY_SERVER,
+                       "Remove DataSetReader failed. DataSetReader is frozen.");
```

Comment:
I would prefer 'Subscriber configuration is frozen' (for all the logs here) because the user can't modify the freeze of single entities.

Reply:
Yes done.